console/src/main/scripts/plugins/metrics/ts/app-details/appServerDatasourcesDetails.ts

```diff
@@ -240,6 +224,27 @@ module HawkularMetrics {
       this.getDrivers();
     }
 
+    private getAlerts(res:any):void {
```

Comment:
Yes, this is actually not new, it's an artifact of the original code that I've been working on.  Same for the 'data' below.  I can look to see if I can improve it...